components/create_post/create_post.jsx

```diff
@@ -434,26 +434,25 @@ export default class CreatePost extends React.Component {
             command === 'dnd' || command === 'offline';
     };
 
-    handleSubmit = (e) => {
+    handleSubmit = async (e) => {
         const {
             currentChannel: updateChannel,
             userIsOutOfOffice,
         } = this.props;
 
         if (this.props.isTimezoneEnabled) {
-            this.props.actions.getChannelTimezones(this.props.currentChannel.id).then(
-                (data) => {
-                    if (data.data) {
-                        this.setState({channelMembersCount: data.data.length});
-                    } else {
-                        this.setState({channelMembersCount: 0});
-                    }
-                }
-            );
+            const {data} = await this.props.actions.getChannelTimezones(this.props.currentChannel.id);
```

Comment:
This async call will now delay all posts, even if `containsAtChannel` is false on line 456 below. Should we only invoke this when we know we'll need the data?

Reply:
ah, thats true, will check the `containsAtChannel` as well

Reply:
PTAL @lieut-data